src/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobBuilder.cs

```diff
@@ -305,7 +305,7 @@ public byte[] ToArray(int start, int byteCount)
                         break;
                     }
 
-                    Buffer.BlockCopy(chunk._buffer, Math.Max(start - chunkStartPosition, 0), result, resultOffset, bytesToCopy);
+                    Array.Copy(chunk._buffer, Math.Max(start - chunkStartPosition, 0), result, resultOffset, bytesToCopy);
```

Comment:
I guess that explains the whitespace diffs too. Feel free to ignore that feedback if it's too cumbersome with the syncing between code bases.